### ISSUE TITLE
Navigate to previous or next short without animation with W and S doesn't work

### ISSUE THREAD

**Issue by bojacc** [ynshung/better-yt-shorts#142]
Hi, in version 3.0.1, the shorts are not getting navigated without animation, the shortcut doesn't work out of the box however if we do set up a Keybind for next/previous shorts, it gets navigated with the default animation. 
I tried with Edge browser (Version 117.0.2045.5) and Firefox browser (118.0.1).  
Not sure if it is just me or other people are facing it as well, although not a dealbreaker though.

I installed a previous version of extension on firefox browser, all other features were broken however the W/S without animation switching was working flawlessly.

**Comment by adsuth** [comment]
which version specifically had it without the animation? I know that we changed the way it skips a short to synthetically click the next button instead of instantly scrolling down because we were seeing issues on mac with the latter

**Comment by ynshung** [comment]
It seems like clicking the previous/next button now also triggers the animation and it's probably a change from youtube themselves. If there isn't any way to skip shorts without animation, we'll either leave the keybinds and remove the part we mention about skipping without animation or remove it altogether

**Comment by bojacc** [comment]
> It seems like clicking the previous/next button now also triggers the animation and it's probably a change from youtube themselves. If there isn't any way to skip shorts without animation, we'll either leave the keybinds and remove the part we mention about skipping without animation or remove it altogether

Yes it could be, but i just posted this to confirm with others if others were also experiencing this issue. 
It was actually pretty snappy that non-animation switching to next/previous reel, i liked it haha.

**Comment by bojacc** [comment]
> which version specifically had it without the animation? I know that we changed the way it skips a short to synthetically click the next button instead of instantly scrolling down because we were seeing issues on mac with the latter

2.7.2

**Comment by adsuth** [comment]
Gave it a check, and 2.7.2 was still doing the old method of moving between shorts.

Could potentially add a "no animation" option that uses that old method, but again, that may not work on other devices

**Comment by bojacc** [comment]
> Gave it a check, and 2.7.2 was still doing the old method of moving between shorts.
> 
> Could potentially add a "no animation" option that uses that old method, but again, that may not work on other devices

Thank you for the prompt response. I would really appreciate if it can be implemented, however if there are a lot of variables involved, i dont mind the short switching animation much. Thank you for all the work done on this extension. x)

**Comment by bojacc** [comment]
Hi, 
just wanna follow up with an something I observed.

I recently found this extension in microsoft edge extensions store and the current version is v2.8.3
And the W/S navigation is working here without the animation and I superlike it. Autoplay and most of other features are not available but I guess for now I can live with it.
So here's a request, could you please link me to the unpacked version of v2.8.3 so that this extension doesn't auto-update and it disables that feature of no animation navigation?

**Comment by ynshung** [comment]
Hi there @bojacc, you could navigate to the v2 branch [here](https://github.com/ynshung/better-yt-shorts/tree/v2) and download the repository as ZIP then load them unpacked from there

**Comment by bojacc** [comment]
Thank you. Appreciate the help.

Any plans on keeping this feature alive in v3?

**Comment by ynshung** [comment]
Definitely will try to search a workaround for this. It would be great if anyone could find the solution for this
